help/user-guide/browsing.md

```diff
@@ -24,3 +24,45 @@ Alternatively, a condensed view and a table view is available, showing more resu
 The columns to be shown by the table view can be customised, adding/removing columns as well as choosing their order.
 
 ![Table view configuration](/images/user-guide/browsing_table_config.png){:.screenshot}
+
+# Browsing Samples
+You can browse Samples registered in SEEK by Sample Type or by Templates.
+* From any page, click on Browse from the top navigation bar
+* Select Samples from the dropdown menu
+
+## Browse Samples by Sample Type
+Browsing Samples by Sample Type allows you to browse the samples generated by one Sample Type. Samples can be browsed based on one Sample Type at the time.
+* Select the Project(s) to which the Sample Type of interest must belong. You can select Projects among
+  * Your Projects: these are the Projects you are a member of that have sample types defined
+  * Other Projects: these are the Projects you are not a member of that have sample types defined
+* Scroll down to see the list of all selected Sample Types. Click on the Sample Type of your interest
+* Click on View samples button (visible only if at least one Sample has been created with the selected Sample Type by any user) on the top right corner of the page
+* Use the search boxes in each column to browse samples
+* Click on Export table to export the search result as spreadsheet
+
+## Browse Samples by Experiment Sample Template
+Browsing Samples by Experiment Sample Template allows you to query the samples of interest based on Experiment Sample Template, its attributes and the attribute’s values of the samples of interest.
```

Comment:
Add note that this only works if the ISA-JSON compliance feature is enabled in the admin panel.